packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <div
+    data-cy="debug"
+    class="flex flex-col pb-24px gap-16px"
+  >
+    <div
+      :data-cy="`debug-header-${debug.id}`"
+      class="grid px-24px w-full overflow-hidden flex items-center border gap-y-2 py-24px"
+    >
+      <ul
+        :data-cy="'header-top'"
+        class="flex self-stretch items-center gap-x-2 flex-row whitespace-nowrap"
+      >
+        <li
+          v-if="debug?.commitInfo?.summary"
+          class="font-medium text-lg text-gray-900"
+          data-cy="debug-test-summary"
+        >
+          {{ debug.commitInfo.summary }}
+        </li>
+        <div
+          v-if="defaultscommitInfo.runNumber && commitsAhead"
+          class="border rounded border-gray-100 items-center text-sm h-6"
+          data-cy="debug-runCommit-info"
+        >
+          <span
+            class="font-medium text-gray-700 border-r-1px px-2 mx-px items-center"
+            data-cy="debug-runNumber"
+          >
+            {{ defaultscommitInfo.runNumber }}
+          </span>
+          <span
+            class="font-normal text-orange-500 px-2 mx-px items-center"
+            data-cy="debug-commitsAhead"
+          >
+            {{ commitsAhead }}
+          </span>
+        </div>
+        <li class="-mt-8px text-lg text-gray-400">
+          .
+        </li>
+        <li class="font-normal text-sm text-indigo-500">
+          <ExternalLink
+            :data-cy="'debug-header-dashboard-link'"
+            :href="debug.url || '#'"
+            :use-default-hocus="false"
+          >
+            <span class="sr-only">Dashboard Link:</span> View in the dashboard
+          </ExternalLink>
+        </li>
+      </ul>
+      <ul
+        :data-cy="'metadata'"
+        class="flex flex-wrap gap-x-2 items-center whitespace-nowrap children:flex children:items-center font-normal text-sm text-gray-700"
+      >
+        <li
+          :data-cy="'debug-results'"
+        >
+          <DebugResults
+            :gql="props.gql"
+          />
+        </li>
+        <li
+          v-if="debug?.commitInfo?.branch"
+          data-cy="debug-header-branch"
+        >
+          <i-cy-tech-branch-h_x16 class="mr-1 icon-dark-gray-300 mr-9px" />
```

Comment:
margins/paddings are normally factors of 4 (normally 8), so seeing `mr-9px` makes me think it should be `mr-8px`